src/components/Icons/Social/Gift.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { Path } from 'react-native-svg';
+import { colors } from '@magnetis/astro-galaxy-tokens';
+
+import BaseIcon from '../BaseIcon';
+import { getFill, getSize, getViewBox } from '../utils';
+import type { IconProps } from '../types';
+
+function Gift({
+  id = 'GiftIcon',
+  color = colors.space100,
+  viewBox = getViewBox(),
+  width = getSize(),
+  height = getSize(),
+  ...props
+}: IconProps) {
+  const fill = React.useMemo(() => getFill({ gradient: props.gradient, color, id }), [color, props.gradient, id]);
+
+  return (
+    <BaseIcon id={id} color={color} width={width} height={height} viewBox={viewBox} {...props}>
+      <Path
+        fill-Rule="evenodd"
+        clip-Rule="evenodd"
```

Comment:
🔧 Missing fix props case here and below
```suggestion
        fillRule="evenodd"
        clipRule="evenodd"
```